backend/LexBoxApi/Controllers/UserController.cs

```diff
@@ -129,7 +122,7 @@ public async Task<ActionResult<LexAuthUser>> AcceptEmailInvitation(RegisterAccou
         return Ok(user);
     }
 
-    private User CreateUserEntity(RegisterAccountInput input, bool emailVerified, Guid? creatorId = null)
+    private User CreateUserEntity(RegisterAccountInput input, bool emailVerified, LexAuthUser? jwtUser, Guid? creatorId = null)
```

Comment:
you can remove the `emailVerified` parameter as the value is being ignored anyway